internal/documentdb/genwrap/convert.go

```diff
@@ -283,7 +273,7 @@ func (c *Converter) handleFunctionOverloading(f *convertedRoutine) {
 	}
 
 	if !skip && strings.Contains(f.Comment, "documentdb_core.bson,") {
```

Comment:
Why does this method silently skip some functions without using the unique name or, at least, log that something is being skipped?

Reply:
Good find, function overloading is handled in `Convert()` function, this was residue need removing. Added a test for this case :+1: